elastic/kibana
# [SecuritySolution] Missing reset button when there is no data to display

## Summary
issue: https://github.com/elastic/kibana/issues/149828

Please enable feature flag: chartEmbeddablesEnabled

Steps to verify:
1. Go to alerts trend or table chart
2. Delete one or more group by fields
3. You should see chart actions available when there is no data to display
4. Clicking on the reset group by field action, the group by fields should be reset

https://user-images.githubusercontent.com/6295984/215500707-4b7f8076-3e63-4664-beb4-8e5b8fde9f1c.mov



### Checklist

Delete any items that are not applicable to this PR.
